index.js

```diff
@@ -262,8 +262,10 @@ alexa.session = function(session) {
     this.details = {
```

Comment:
Yep, that would be the right thing to do, and is future-proof. I think since we're introducing a backward-incompatible change that makes sense.